html-css/week1/index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="styles.css">
+        <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" rel="stylesheet" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Wasim's Portfolio</title>
+    </head>
+
+    <body>
+        <nav id="navbar" class="nav">
+            <ul class=>
+                <li><a href="#welcome-section">About</a></li>
+                <li><a href="#projects">Projects</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+
+        <header>
+            <h1>SAYYED WASIM HUSSAIN BUKHARI</h1>
+            <p>Full Stack Developer (Trainee)</p>
+        </header>
+
+        <section id="welcome-section">
+            <div class="welcome-section">
+                <div class="intro">
+                    <p> I can provide solution in transforming your business to e commerce.
+                        You can get design, development and
+                        back end support at the same place.
+                    </p>
+                    <p> Have a glimpse of my work
+                        <a href="#projects"> here<span class="material-symbols-outlined">highlight_mouse_cursor</span></a>
+                    </p>
+                </div>
+                <div class="image">
+                    <img src="https://ca.slack-edge.com/T428UGBJA-U086WDM2328-54b75f219606-512" alt="Wasim Photo">
+                </div>
```

Comment:
You can reduce the nesting by moving the class directly to the image tag.
```suggestion
            <img src="https://ca.slack-edge.com/T428UGBJA-U086WDM2328-54b75f219606-512" alt="Wasim Photo"  class="image" />
```

Reply:
This one if I remove the div and put class in <img> it changes the settings of the picture as I  need. I can not control the same size for text and image.